modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -82,26 +88,48 @@ pullSecret: '<pull_secret>' <6>
+<3> Depending on your infrastructure, you can select either `arm64` or `amd64`.
 <4> Set `OCI` as the external platform, so that {product-title} can integrate with {oci}.
 <5> Specify your SSH public key.
 <6> The pull secret that you need for authenticate purposes when downloading container images for {product-title} components and services, such as Quay.io. See link:https://console.redhat.com/openshift/install/pull-secret[Install {product-title} 4] from the {hybrid-console}.
 
-. Create a directory on your local system named `openshift`.
+. Create a directory on your local system named `openshift`. This must be a subdirectory of the installation directory.
 +
 [IMPORTANT]
 ====
 Do not move the `install-config.yaml` and `agent-config.yaml` configuration files to the `openshift` directory.
 ====
 
-. Complete the steps in the link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files["Configuration Files"] section of the _Oracle_ documentation to download {oci-ccm-full} and {oci-csi-full} manifests as an archive file and save the archive file in your `openshift` directory. You need the {oci-ccm} manifests for deploying the {oci-ccm} during cluster installation so that {product-title} can connect to the external {oci} platform. You need the {oci-csi} custom manifests for deploying the {oci-csi} driver during cluster installation so that {product-title} can claim required objects from {oci}.
+. If you used a stack to provision OCI infrastructure resources: Copy and paste the `dynamic_custom_manifest` output of the OCI stack into a file titled `manifest.yaml` and save the file in the `openshift` directory.
+
+. If you did not use a stack to provision OCI infrastructure resources: Download and prepare custom manifests to create an Agent ISO image:
+
+.. Go to link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files[Configuration Files] (Oracle documentation) and follow the link to the custom manifests directory on GitHub.
+
+.. Copy the contents of the `condensed-manifest.yml` file and save it locally to a file in the `openshift` directory.
+
+.. In the `condensed-manifest.yml` file, update the sections marked with `TODO` to specify the compartment {ocid}, VCN {ocid}, subnet {ocid} from the load balancer, and the security lists {ocid}.
 
-. Access the custom manifest files that are provided in the link:https://docs.oracle.com/iaas/Content/openshift-on-oci/install-prereq.htm#install-configuration-files["Configuration Files"] section of the _Oracle_ documentation.
+. Configure the `agent-config.yaml` configuration file to meet your organization's requirements.
 +
-.. Change the `oci-cloud-controller-manager` secret that is defined in the `oci-ccm.yml` configuration file to match your organization's region, compartment {ocid}, VCN {ocid}, and the subnet {ocid} from the load balancer.
+.`agent-config.yaml` file that sets values for an IPv4 formatted network.
```

Comment:
Doesn't need a `.`, probably should call it a "sample", and I _think_ you could shorten it with no impact to understanding 
```suggestion
.Sample `agent-config.yaml` file for an IPv4 network
```

Reply:
Good call, not my original writing (i only switched the order of steps for accuracy) so I'm happy to trim out any excess